README.md

```diff
@@ -1,116 +1,76 @@
-    case "selections":
-        fmt.Println("--Selections MFA--")
-        fmt.Println("Question:", mfaRes.Selections[1].Question)
-        fmt.Println("Answers:", mfaRes.Selections[1].Answers)
-    }
-
-    postRes2, mfaRes2, err := client.AuthStepSendMethod(mfaRes.AccessToken, "type", "email")
-    if err != nil {
-        fmt.Println("Error submitting send_method", err)
-    }
-    fmt.Println(mfaRes2, postRes2)
-
-    postRes2, mfaRes2, err = client.AuthStep(mfaRes.AccessToken, "tomato")
-    if err != nil {
-        fmt.Println("Error submitting mfa", err)
-    } else {
-        fmt.Println(mfaRes2, postRes2)
-    }
-} else {
-    fmt.Println(postRes.Accounts)
-    fmt.Println("Auth Get")
-    fmt.Println(client.AuthGet("test_bofa"))
-
-    fmt.Println("Auth DELETE")
-    fmt.Println(client.AuthDelete("test_bofa"))
-}
-```
+| API version | plaid-ruby release |
+| ----------- | ------------------ |
+| [`2018-05-22`][api-version-2018-05-22] (**latest**) | `6.x.x` |
+| `2017-03-01` | `5.x.x` |
 
-### Plaid Link Exchange Token Process
+For information about what has changed between versions and how to update your integration, head to the [version changelog][version-changelog].
 
-Exchange a [Plaid Link][1] `public_token` for an API `access_token`:
+## Documentation
 
-```go
-client := plaid.NewClient("test_id", "test_secret", plaid.Tartan)
-
-// POST /exchange_token
-postRes, err := client.ExchangeToken(public_token)
-if err != nil {
-    fmt.Println(err)
-} else {
-    // Use the returned Plaid API access_token to retrieve
-    // account information.
-    fmt.Println(postRes.AccessToken)
-    fmt.Println("Auth Get")
-    fmt.Println(client.AuthGet(postRes.AccessToken))
-}
-```
+The module supports all Plaid API endpoints.
 
-With the [Plaid + Stripe ACH integration][2], exchange a Link `public_token`
-and `account_id` for an API `access_token` and Stripe `bank_account_token`:
+GoDoc: [![GoDoc](https://godoc.org/github.com/plaid/plaid-go?status.svg)](https://godoc.org/github.com/plaid/plaid-go/plaid)
 
-```go
-client := plaid.NewClient(CLIENT_ID, SECRET, plaid.Tartan)
-
-// POST /exchange_token
-postRes, err := client.ExchangeTokenAccount(public_token, account_id)
-if err != nil {
-    fmt.Println(err)
-} else {
-    // Use the returned Plaid access_token to make Plaid API requests and the
-    // Stripe bank account token to make Stripe ACH API requests.
-    fmt.Println(postRes.AccessToken)
-    fmt.Println(postRes.BankAccountToken)
-}
-```
+## Getting Started
+
+### Calling Endpoints
+
+To call an endpoint you must create a `Client` object.
 
-### Querying a category
 ```go
-// GET /categories/13001001
-category, err := plaid.GetCategory(plaid.Tartan, "13001001")
-if err != nil {
-    fmt.Println(err)
-} else {
-    fmt.Println("category", category.ID, "is", strings.Join(category.Hierarchy, ", "))
-}
+import (
+    "os"
+
+    "github.com/plaid/plaid-go/plaid"
+)
+
+client := plaid.NewClient(
```

Comment:
we'll need to update the documentation here if we decide to restructure the constructor as i suggested.

Reply:
⚡️